src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -531,9 +536,12 @@ class VAMap extends Component {
                 attribution='Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, \
                   <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, \
                   Imagery © <a href="http://mapbox.com">Mapbox</a>' />
-              <FeatureGroup ref="facilityMarkers">
-                {this.renderFacilityMarkers()}
-              </FeatureGroup>
+              {facilityLocatorMarkers.length > 0 &&
+                <FeatureGroup
+                  ref="facilityMarkers">
+                  {facilityLocatorMarkers}
+                </FeatureGroup>
```

Comment:
An error was being thrown in `FeatureGroup ` if `renderFacilityMarkers` returned an empty array.

Reply:
So moving the building of the markers array up above to the variable was the fix for this? Weird it would throw.

Reply:
The fix was checking the length of the markers array prior to rendering the `FeatureGroup`, but I moved the variable above so `renderFacilityMarkers` only needed to be called once.